.idea/codeStyles/Project.xml

```diff
@@ -157,4 +162,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```

Comment:
Revert this file. We do not want these changes.

Reply:
@PrarabdhGarg This file is still here. We do not want to make any changes to this file. So please revert back this file to its original code.